src/gc/env/gcenv.interlocked.inl

```diff
@@ -116,6 +116,37 @@ __forceinline T Interlocked::ExchangeAdd(T volatile *addend, T value)
 #endif
 }
 
+template <typename T>
+__forceinline T Interlocked::ExchangeAdd64 (T volatile* addend, T value)
```

Comment:
```suggestion
__forceinline T Interlocked::ExchangeAdd64 (T volatile* addend, T value)
```

(I am not commenting on all these - it would be nice to have consistent formatting within a file at least.)